core/Command/App/ListApps.php

```diff
@@ -50,6 +50,18 @@ protected function configure() {
 		$this
 			->setName('app:list')
 			->setDescription('List all available apps.')
+			->addOption(
+				'enabled',
+				'e',
+				InputOption::VALUE_NONE,
+				'Only display enabled apps.'
+			)
+			->addOption(
+				'disabled',
+				'd',
+				InputOption::VALUE_NONE,
+				'Only display disabled apps.'
```

Comment:
@settermjd Are you suggesting to add words here to indicate that the app version might be displayed? e.g.
```suggestion
				'Only display disabled apps. If the app was previously enabled, the app version is also displayed. '
```

Reply:
@settermjd ping

Reply:
Hey! That's what I understood after reading through the PR, yes.